data/khushbumaheshwarii.json

```diff
@@ -0,0 +1,20 @@
+	{
+  "name": "Khushbu Maheshwari",
+  "displayStarsPublic":true,
```

Comment:
please remove "displayStarsPublic":true

Reply:
Thank you for reviewing @harshsingh32 but this comment was already made earlier [here](https://github.com/EddieHubCommunity/LinkFree/pull/6186#pullrequestreview-1390319158), repeating comments does not help, it is spam and scares people off from contributing, please read existing comments on the timeline first

Reply:
I'll read existing comments on the timeline first.